docs/ai-agents/README.md

```diff
@@ -1,104 +1,81 @@
----
-products: embedded
----
+import Taxonomy from "@site/static/_taxonomy_of_data_movement.md";
 
-# AI Agents
+# Agent engine
 
-Airbyte provides multiple tools to help you build data applications.
+Airbyte's Agent engine is a set of tools to help you automate, understand, move, and work with your data in coordination with AI agents. Some of these tools are standalone open source solutions, and others are pay solutions built on top of Airbyte Cloud.
 
-- **MCP Servers**: Airbyte provides multiple MCP (Model Context Protocol) servers for different use cases:
-  - [**PyAirbyte MCP**](#pyairbyte-mcp): Local MCP server for managing Airbyte connectors through AI assistants.
-  - [**Connector Builder MCP**](#connector-builder-mcp): AI-assisted connector development - _**coming soon!**_
-- **Airbyte Embedded Widget**: App development teams who have signed up for Airbyte Embedded and are looking to get started onboarding customers using the Embedded Widget can follow the get started guide at the bottom of this page, which will step you through a complete sample onboarding app.
-- **Authentication Proxies**: Connect safely to third party APIs using Airbyte's Authentication Proxies.
+The platform is a solution for all types of audiences, from AI engineers who are deploying agents in large enterprises down to individual founders who need real-time context and action in their platforms.
 
-## Prerequisites
+- **MCP Servers**: Model Context Protocol (MCP) servers for different use cases. The [**PyAirbyte MCP**](#pyairbyte-mcp) is a local MCP server for managing Airbyte connectors through AI assistants. The [**Connector Builder MCP (coming soon)**](#connector-builder-mcp) is an AI-assisted connector development.
```

Comment:
the pyairbyte mcp isn't part of the agent engine. maybe we want to start talking about it as the replication mcp?